contrib/tnitter/Makefile

```diff
@@ -1,3 +1,24 @@
-all:
+SERVER ?= localhost:8080
```

Comment:
The android-release use the real server, as well as the config for xymus.net. The SERVER var is only for local tests and deployment.